todo_app/app.py

```diff
@@ -18,6 +26,31 @@ def new_item():
     add_item(field_name)
     return redirect(url_for('index'))
 
+@app.route('/trello')
+def get_cards():
+    params = {"key": KEY, "token": TOKEN}
+    cards_in_board = requests.get("https://api.trello.com/1/lists/" + TODOID + "/cards", params = params)
```

Comment:
This looks like a cards in list url rather than a cards in board url.

Reply:
changed cards_in_board to cards_in_list